Formula/j/jaguar.rb

```diff
@@ -0,0 +1,38 @@
+class Jaguar < Formula
+  desc "Live reloading for your ESP32"
+  homepage "https://toitlang.org/"
+  url "https://github.com/toitlang/jaguar/archive/refs/tags/v1.48.0.tar.gz"
+  sha256 "fb5a329ef5c166a791c84adf4a9c3c8003c974fe58384d428bde84baf1ec7493"
+  license "MIT"
+  head "https://github.com/toitlang/jaguar.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.buildDate=#{time.iso8601}
+      -X main.buildMode=release
+    ]
+    system "go", "build", *std_go_args(output: bin/"jag", ldflags:), "./cmd/jag"
```

Comment:
let's order in this way

```suggestion
    system "go", "build", *std_go_args(ldflags:, output: bin/"jag"), "./cmd/jag"
```